nvflare/fuel/f3/drivers/driver_manager.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2021-2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import importlib
+import inspect
+import logging
+import os
+from typing import Optional, Type
+
+from nvflare.fuel.f3.comm_error import CommError
+from nvflare.fuel.f3.drivers.driver import Driver
+
+log = logging.getLogger(__name__)
+
+
+class DriverManager:
```

Comment:
should we move this into upper layer?
``nvflare/fuel/f3/driver_manager.py``

Reply:
This is not exposed so it's better to stay here.